### PR TITLE
Making the module compatible with the last version of magisk

### PR DESCRIPTION
**Pull request by geekywoman**
In the current state, the module is not appearing in the list of magisk online repos and installing the module manually makes the device stuck into bootloop.

Having the minMagisk version update according to the magisk templates guidelines here: https://topjohnwu.github.io/Magisk/guides.html#magisk-module-template seems to solve the issue.

### CODE REVIEW

**Comment by AndroPlus-org** [comment]
No, he won't add this module because of "A module that only uses system.prop to modify system props.".
Also your pull request is not enough for updating minMagisk version.

**Comment by geekywoman** [comment]
Can you be more specific about this "A module that only uses system.prop to modify system props."?

Alright, I will use it as a custom module that I will install manually with the last version of magisk then. That's a pity.

**Comment by AndroPlus-org** [comment]
There is no detail info, but it's in "What Will NOT be Approved" section:
https://github.com/Magisk-Modules-Repo/submission
This requirements are not a good thing since simple DNS modules are listed while my modules are not...lol

**Comment by geekywoman** [comment]
I am totally new to this module thing, but is there any alternative to making this "hide navigation bar" approach other than having a "system.prop to modify system props"?

I discovered your module for the first time after rooting some lenovo tab4 with the system partition being read only and I was not able to update the build.prop file myself without the +W settings. It really help me accomplish what I was looking for at the time but with a way holder version of magisk.

Anyway, long story short:
I think you module is very useful to the community, and it would be a pity to not see it anymore in the module list of magisk. 

I am a complete noob in the module-magisk-environement but I am an Android dev and I like to play with the android system. So if another solution is existing out there, and in the case you need any help  accomplishing it you know were to find me.

**Comment by geekywoman** [comment]
Well, if you are not gonna update it I would suggest to at least warn people that the module is not longer supported and won't work. Installing it manually will result having the device stuck at the boot.

**Comment by AndroPlus-org** [comment]
Maybe it's device-specific bug since it's working on my device, but I'll close this repo as I won't maintain much.
